packages/client/src/lib.js

```diff
@@ -94,15 +108,48 @@ class NFTStorage {
   }
   /**
    * @param {API.Service} service
-   * @param {Blob} blob
-   * @returns {Promise<API.CIDString>}
+   * @param {Blob|CarReader} carReader
+   * @param {{onStoredChunk?: (cid: API.CIDString) => void}} [options]
```

Comment:
The CID will always be the same so not really useful...

```suggestion
   * @param {{onStoredChunk?: (size: number) => void}} [options]
```